### DISCUSSION TITLE
start supporting pacman

### DISCUSSION THREAD

**Discussion by Maxython** [termux-user-repository/tur#84] [Ideas]
Hi all, as the *founder* and *director* of the termux-pacman organization, I'm interested in starting to maintain your repository for the pacman package manager.

**Comment by Maxython** [comment]
![Screenshot_20220916_150956](https://user-images.githubusercontent.com/60917834/190636223-48621ab0-03bc-4e89-81ec-6d333193de49.jpg)

**Comment by kcubeterm** [comment]
That's awesome.

**Comment by jstetten** [comment]
@Maxython, checking out your project now. Very interesting.

The last official release is from more than a year ago and in beta form. Do you have any plans to put out a newer update sometime? (No diss, just wondering...it's very impressive.)

**Reply by Maxython** [reply]
The [pacman-for-termux](https://github.com/Maxython/pacman-for-termux) repository currently plays the role of informing the user, so it will no longer contain updates and new releases of pacman. But this does not mean that it will remain old, all work with the source code and updating pacman have been moved to the [termux-packages](https://github.com/termux/termux-packages/tree/master/packages/pacman) repository.

**Reply by jstetten** [reply]
Ah, got it. I didn't realize your repo had been integrated. Thanks!
